In [1]:
import numpy as np
np.random.seed(1)

# Generate random numbers in python

x = np.random.randint(0,50,1000)
x

array([37, 43, 12,  8,  9, 11,  5, 15,  0, 16,  1, 12,  7, 45,  6, 25, 20,
       37, 18, 20, 11, 42, 28, 29, 14,  4, 23, 23, 41, 49, 30, 32, 22, 13,
       41,  9,  7, 22,  1,  0, 17,  8, 24, 13, 47, 42,  8, 30,  7,  3,  6,
       21, 49,  3,  4, 24, 49, 43, 12, 26, 16, 45, 41, 18, 15,  0,  4, 25,
       47, 34, 23,  7, 26, 25, 40, 22,  9,  3, 39, 23, 36, 27, 37, 19, 38,
        8, 32, 34, 10, 23, 15, 47, 23, 25,  7, 28, 10, 46, 32, 24, 23,  1,
       49, 13,  3,  0, 13,  6, 21,  6,  2, 12, 27, 21, 11,  7, 13,  8, 11,
       12, 43, 20, 30, 36, 39,  7, 45,  4, 48, 18, 32, 13, 10, 23, 17,  7,
       24, 10, 28, 20, 32, 12,  1, 30, 41, 24, 18, 33,  2, 44, 28, 39, 34,
       10, 32, 41, 18, 22,  6,  2,  7, 39, 48, 15,  5, 17, 42, 20, 48, 22,
       37, 13, 49, 33, 20, 10, 32, 25, 21, 39, 32, 19, 25, 49, 14, 13, 20,
       43,  6, 13, 15, 24,  9,  2,  7,  5, 36, 37, 21, 40, 13, 27, 49, 49,
       47, 13, 40, 14, 45, 23, 16, 28, 42, 45,  3,  2, 14, 46,  0, 29, 11,
       35, 29, 33,  2, 20

In [2]:
y = x +np.random.normal(0,10,1000)
y

array([ 20.91943595,  50.63680457,  26.1899858 ,   4.2567297 ,
         4.76682859,  -1.41868013,   0.13974066,  12.68595762,
        -2.4644001 ,  23.67966704,  -5.26864647,  44.705657  ,
         3.929594  ,  41.15251149,   6.15189145,  16.87380256,
        12.12197948,  23.85785176,  19.98609381,  17.29294368,
        13.81247183,  29.45979256,  29.23125049,  15.30400214,
         7.32581754,   8.57716492,   8.37879784,  25.63585484,
        49.04095427,  58.34967036,  32.56479127,  29.70811918,
         1.18361352,  25.66966811,  49.05104488,  18.60306431,
        11.34949454,  28.80212162,  19.68884019,   1.50705284,
        11.31341917,  20.74937886,  31.50733465,  21.09738523,
        32.61895129,  58.59052608,  11.11072572,  32.15714494,
         2.25252158,  -0.26341235,  -1.10116519,  10.9632299 ,
        50.14247927,  13.13755659,  -3.20034693,  19.75343555,
        57.20068833,  27.88677104,   7.24892855,   6.40492564,
        28.57645233,  42.85817494,  22.3029112 ,   9.07

In [3]:
c = np.corrcoef(x,y)
c

array([[1.        , 0.81543901],
       [0.81543901, 1.        ]])

Correllation between x and y = 0.815 -> The correlation between is positive also high correlation exits